### PR TITLE
Add load tests for insert, delete, update operations

### PR DESCRIPTION
**Pull request by kaneeldias**
$subject

Related issue: ballerina-platform/ballerina-standard-library#2270

### CODE REVIEW

**Comment by dilanSachi** [comment]
There was an issue with trigger-load-test workflow when specifying test names. It is fixed in this PR https://github.com/ballerina-platform/module-ballerina-grpc/pull/519.
Kindly fix it in here also

**Comment by kaneeldias** [comment]
> There was an issue with trigger-load-test workflow when specifying test names. It is fixed in this PR [ballerina-platform/module-ballerina-grpc#519](https://github.com/ballerina-platform/module-ballerina-grpc/pull/519). Kindly fix it in here also

Fixed in 65958c7 and b7d2e43

File: load-tests/delete/scripts/http-delete-request.jmx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          <stringProp name="outputFormat">000</stringProp>
+          <boolProp name="perThread">true</boolProp>
+          <stringProp name="randomSeed">1</stringProp>
+          <stringProp name="variableName">random</stringProp>
+        </RandomVariableConfig>
+        <hashTree/>
+      </hashTree>
+    </hashTree>
+  </hashTree>
+</jmeterTestPlan>
```
Comment (kalaiyarasiganeshalingam) [inline load-tests/delete/scripts/http-delete-request.jmx:71]:
```suggestion
</jmeterTestPlan>

```
Please check all the places as well.

Comment (kaneeldias) [reply load-tests/delete/scripts/http-delete-request.jmx]: Fixed in 12d3cbb

**Review by kalaiyarasiganeshalingam — APPROVED** [review]

**Pull request merged**